src/app/containers/play/hooks/fetchVideoUrl.js

```diff
@@ -0,0 +1,13 @@
+import { useQuery } from "@tanstack/react-query"
+import axios from "axios"
+
+const fetchVideoUrl = async (id) => {
+    const response = await axios.get(`/api/movie/${id}`)
+
+    return response.data
+}
+
+export const useVideoUrl = (id) => useQuery({
```

Comment:
If we had a video endpoint, this would be the way. But since we fetch the movie, we could reuse the useMovie hook this time